# Admittance parameters for two mass damper systems
# (the arm and the platform) with params (mass_arm, damping_arm)
# and (mass_platform, damping_platform) respectively, connected by a
# spring-damper with parameters (damping_coupling, stiffness_coupling,
# equilibrium_point_spring)
mass_arm: [0.2,0,0,0,0,0,
           0,0.2,0,0,0,0,
           0,0,0.2,0,0,0,
           0,0,0,0.2,0,0,
           0,0,0,0,0.2,0,
           0,0,0,0,0,0.2]
damping_arm: [7,0,0,0,0,0,
              0,7,0,0,0,0,
              0,0,7,0,0,0,
              0,0,0,7,0,0,
              0,0,0,0,7,0,
              0,0,0,0,0,7]
stiffness_arm: [10,0,0,0,0,0,
                0,10,0,0,0,0,
                0,0,10,0,0,0,
                0,0,0,10,0,0,
                0,0,0,0,10,0,
                0,0,0,0,0,10]
# [position (x, y, z)  orientation (x, y, z, w)]
#equilibrium_point_spring: [0.1,0.4,0.5,-0.593599, -0.566706, -0.369419, 0.435908]
equilibrium_point_spring: [-0.341,-0.103,0.657,-0.49, -0.1520, 0.8077, -0.2835]


workspace_limits: [-0.70, 0.70, -0.7, 0.5, 0.15, 0.75]

arm_max_vel: 1.5
arm_max_acc: 1.0